inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -376,6 +376,45 @@ void FrontEnd::parseLayer(const Model& model, const ie::CNNLayerPtr& layer, cons
     }
 }
 
+void FrontEnd::processTrivialCases(const Model& model) {
+    DataVector trivialInputs, trivialOutputs;
+
+    for (const auto& data : model->datas()) {
+        if (data->usage() == DataUsage::Input || data->usage() == DataUsage::Intermediate || data->usage() == DataUsage::Const) {
```

Comment:
Is it okay to process this way intermediate data objects with no consumers? Looks a bit strange..

Reply:
I process `Intermediate` objects alongside `Input` and `Output` because the input/output conversion has already been applied at this moment, so now we may have unconnected converted input/output which are `Intermediate`.